### PR TITLE
Update third-party-monitoring-integrations.md

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by mashenjun — APPROVED** [review]

File: tidb-cloud/monitor-datadog-integration.md [hunk 1]
```diff
@@ -17,7 +17,9 @@ You can configure TiDB Cloud to send metric data about your TiDB clusters to [Da
 
 ## Limitation
 
-You cannot use the Datadog integration in [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
+1. You cannot use the Datadog integration in [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
+
+2. Datadog Integrations are not available when the cluster status is CREATING, RESTORING, PAUSED, or RESUMING.
```
Comment (qiancai) [inline tidb-cloud/monitor-datadog-integration.md:22]:
```suggestion
- You cannot use the Datadog integration in [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).

- Datadog integrations are not available when the cluster status is **CREATING**, **RESTORING**, **PAUSED**, or **RESUMING**.
```

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 2]
```diff
@@ -17,7 +17,9 @@ This document describes how to configure your Prometheus service to read key met
 
 ## Limitation
 
-You cannot use the Prometheus and Grafana integration in [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
+1. You cannot use the Prometheus and Grafana integration in [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
+
+2. Prometheus and Grafana Integrations are not available when the cluster status is CREATING, RESTORING, PAUSED, or RESUMING.
```
Comment (qiancai) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:22]:
```suggestion
- You cannot use the Prometheus and Grafana integration in [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).

- Prometheus and Grafana integrations are not available when the cluster status is **CREATING**, **RESTORING**, **PAUSED**, or **RESUMING**.
```

File: tidb-cloud/third-party-monitoring-integrations.md [hunk 3]
```diff
@@ -25,6 +21,12 @@ To edit third-party integration settings, you must have the `Owner` access to yo
 
 The available third-party integrations are displayed.
 
+## Limitation
+
+1. For [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta), third-party monitoring integrations are not supported.
+
+2. Third-Party Monitoring Integrations are not available when the cluster status is CREATING, RESTORING, PAUSED, or RESUMING.
```
Comment (qiancai) [inline tidb-cloud/third-party-monitoring-integrations.md:28]:
```suggestion
- For [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta), third-party monitoring integrations are not supported.

- Third-party monitoring integrations are not available when the cluster status is **CREATING**, **RESTORING**, **PAUSED**, or **RESUMING**.
```

**Review by qiancai — APPROVED** [review]
LGTM

File: tidb-cloud/monitor-datadog-integration.md [hunk 4]
```diff
@@ -17,7 +17,9 @@ You can configure TiDB Cloud to send metric data about your TiDB clusters to [Da
 
 ## Limitation
 
-You cannot use the Datadog integration in [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
+- You cannot use the Datadog integration in [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
```
Comment (Oreoxmt) [inline tidb-cloud/monitor-datadog-integration.md:20]:
```suggestion
- You cannot use the Datadog integration in [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters.
```

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 5]
```diff
@@ -17,7 +17,9 @@ This document describes how to configure your Prometheus service to read key met
 
 ## Limitation
 
-You cannot use the Prometheus and Grafana integration in [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
+- You cannot use the Prometheus and Grafana integration in [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
```
Comment (Oreoxmt) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:20]:
```suggestion
- You cannot use the Prometheus and Grafana integration in [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters.
```

File: tidb-cloud/third-party-monitoring-integrations.md [hunk 6]
```diff
@@ -25,6 +21,12 @@ To edit third-party integration settings, you must have the `Owner` access to yo
 
 The available third-party integrations are displayed.
 
+## Limitation
+
+- For [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta), third-party monitoring integrations are not supported.
```
Comment (Oreoxmt) [inline tidb-cloud/third-party-monitoring-integrations.md:26]:
```suggestion
- For [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters, third-party monitoring integrations are not supported.
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**